delivery/meteorsController.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import { fetchMeteorsUseCase } from '../usecases/fetchMeteors.js';
+
+const router = express.Router();
+
+router.get('/meteors', async (req, res) => {
+  try {
+    const meteorList = await fetchMeteorsUseCase();
+    res.json(meteorList);
+  } catch (error) {
+    console.error('Error in /meteors route:', error);
+    res.status(500).json({ error: 'Failed to fetch meteor data' });
```

Comment:
```
next(error)

// error - can be a custom object
// Example
```js
class Exception {
  constructor(code, massage) {
    this.code = code;
    this.massage = massage;
  }
}

...

next(new Exception(500, 'Error message')
```